src/pages/_feature-flags.tsx

```diff
@@ -13,10 +15,26 @@ function FeatureFlags() {
   });
 
   return (
-    <div>
-      <button id="beta-button">Public Betas</button>
+    <div className="h-screen flex items-center justify-center">
+      <Button id="beta-button">Feature Flags</Button>
+      <Button
+        onClick={() => {
+          posthog.getEarlyAccessFeatures((posthogFeatureFlags) => {
```

Comment:
Question: is this so that we can manually control when to get the latest feature flags in a given session?

Reply:
This
- reads the feature flags for a given user e.g. you, as identified by a [`distinct_id`](https://posthog.com/docs/product-analytics/identify)[^1]
- We then use that response to generate our own `feature_flags` and assign that to a cookie

This is because we need a way to allow these values through the cache which the standard posthog implementation doesn't allow.

[^1]: this is an anonymous ID and doesn't follow you around